velox/exec/tests/FunctionSignatureBuilderTest.cpp

```diff
@@ -124,7 +124,7 @@ TEST_F(FunctionSignatureBuilderTest, typeParamTests) {
           .returnType("integer")
           .argumentType("row(T ..., varchar)")
           .build(),
-      "Type doesn't exist: T ...");
+      "Type doesn't exist: ...");
```

Comment:
This error message is now less clear as it is no longer tell us which type is missing. Would it be possible to continue to include the type name?

Reply:
I see. Would it be clearer to change the test case to "row(T, varchar)"?

Reply:
Changed this to "row(..., varchar)" as what we want to test is that we can't have variable arity as part of row parameters.